docs/manual/common/concepts/index.toc

```diff
@@ -5,3 +5,4 @@ Technologies:Lagom component technologies
 BuildConcepts:Lagom build philosophy
 Immutable:Immutable Objects
 ES_CQRS:Event Sourcing and CQRS
+ServiceDiscovery: Service Discovery
```

Comment:
Thanks for pointing this one out. I've been reading a bit of https://github.com/typesafehub/lightbend-markdown/blob/3a4648bccc4f69408d2d9ba5a68256420d2b7a47/plugin/src/main/scala/com/lightbend/markdown/sbt/LightbendMarkdown.scala#L85-L85 and can't locate a report on `index-toc` or a validation. :-(